src/assets/img/icon/calendar.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="11" height="12" viewBox="0 0 11 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-  <path fill-rule="evenodd" clip-rule="evenodd" d="M3.20833 0C2.9552 0 2.75 0.191878 2.75 0.428571V0.857143H0.916667C0.410406 0.857143 0 1.2409 0 1.71429V3.42857V4.28571V11.1429C0 11.6162 0.410405 12 0.916666 12H10.0833C10.5896 12 11 11.6162 11 11.1429V4.28571V3.42857V1.71429C11 1.2409 10.5896 0.857143 10.0833 0.857143H8.25V0.428571C8.25 0.191878 8.0448 0 7.79167 0C7.53854 0 7.33333 0.191878 7.33333 0.428571V0.857143H3.66667V0.428571C3.66667 0.191878 3.46146 0 3.20833 0ZM10.0833 3.42857V1.71429H8.25V2.14286C8.25 2.37955 8.0448 2.57143 7.79167 2.57143C7.53854 2.57143 7.33333 2.37955 7.33333 2.14286V1.71429H3.66667V2.14286C3.66667 2.37955 3.46146 2.57143 3.20833 2.57143C2.9552 2.57143 2.75 2.37955 2.75 2.14286V1.71429H0.916667L0.916667 3.42857H10.0833ZM0.916667 4.28571L0.916666 11.1429H10.0833V4.28571H0.916667Z" fill="#BEBEBE"/>
+  <path style="fill: #13AA57" fill-rule="evenodd" clip-rule="evenodd" d="M3.20833 0C2.9552 0 2.75 0.191878 2.75 0.428571V0.857143H0.916667C0.410406 0.857143 0 1.2409 0 1.71429V3.42857V4.28571V11.1429C0 11.6162 0.410405 12 0.916666 12H10.0833C10.5896 12 11 11.6162 11 11.1429V4.28571V3.42857V1.71429C11 1.2409 10.5896 0.857143 10.0833 0.857143H8.25V0.428571C8.25 0.191878 8.0448 0 7.79167 0C7.53854 0 7.33333 0.191878 7.33333 0.428571V0.857143H3.66667V0.428571C3.66667 0.191878 3.46146 0 3.20833 0ZM10.0833 3.42857V1.71429H8.25V2.14286C8.25 2.37955 8.0448 2.57143 7.79167 2.57143C7.53854 2.57143 7.33333 2.37955 7.33333 2.14286V1.71429H3.66667V2.14286C3.66667 2.37955 3.46146 2.57143 3.20833 2.57143C2.9552 2.57143 2.75 2.37955 2.75 2.14286V1.71429H0.916667L0.916667 3.42857H10.0833ZM0.916667 4.28571L0.916666 11.1429H10.0833V4.28571H0.916667Z" fill="#BEBEBE"/>
```

Comment:
not sure this is the right way of doing it. it should be another image.

Reply:
I didnt find the same image with that color. So I decided to change color using style